tagger/bad_edge.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"cloud.google.com/go/datastore"
+	"google.golang.org/api/iterator"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+var badRevisions = []string{
+	"032cd9cc63",
+	"064f51c50e",
+	"6e9693d269",
+	"c583bcd7eb",
+	"5d4871b4b8",
+	"8295368c82",
+	"75b0f336c5",
+	"0cc29e423f",
+	"66aabf66f5",
+	"71c05e7131",
+	"b622fea47d",
+	"f5832ccdb3",
+	"09a1d43536",
+	"f3e2b41349",
+	"2ed748f10d",
+	"d44bc3ed38",
+	"168de7c332",
+	"5acd3bcf66",
+	"99e577f57c",
+}
+
+var (
+	projectID = flag.String("project",
+		"wptdashboard-staging",
```

Comment:
How about prod?

Reply:
It can be overridden by a CLI flag.

Reply:
^ yah, i.e. `go run tagger/bad-edge.go -project=wptdashboard`, which I already have done.

Reply:
Okay, not an oversight :)